src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -36,17 +41,27 @@ public static PreSubmitCallbackResponse<SscsCaseData> processPostponementRequest
                                                                                                              GenerateFile generateFile,
                                                                                                              String templateId) {
 
+        log.debug("Executing processPostponementRequestPdfAndSetPreviewDocument for caseId: {}", sscsCaseData.getCcdCaseId());
         final String requestDetails = sscsCaseData.getPostponementRequest().getPostponementRequestDetails();
 
         if (isBlank(requestDetails)) {
             response.addError("Please enter request details to generate a postponement request document");
             return response;
         }
 
+        String ppHearingVenue = sscsCaseData.getPostponementRequest().getPostponementRequestHearingVenue();
+        LocalDate prHearingDate = LocalDateTime.parse(sscsCaseData.getPostponementRequest().getPostponementRequestHearingDateAndTime()).toLocalDate();
```

Comment:
Avoid using short names, had to think about what prHearingDate meant. I take it that you were told to use the information from the postponement request?

Reply:
I wasn't told to use the data from postponement request. But figured out that these data will be available during the Postponement Request step.

Reply:
It's not that bad... I think we can live with those variable names not being completely expanded.